pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.de-de.md

```diff
@@ -38,79 +43,41 @@ details[open]>summary::before {
-| **Step 12** | ❌           | **HCX**                                  | **No HCX usage in vSphere  on OVHcloud.**                              | - If so it cannot be migrated to VCD.                                                                                                                                                                           |                                                                                                                                                                                |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
-
-///
-
-### FAQ - Frequently asked questions
-
-#### In what order should I perform my checks to migrate to VCD?
-
-Looking at the table above, you will notice that a number of checks need to be performed. We recommend launching these pre-checks for your Dedicated Cloud from top to bottom, as suggested in the step column. You can print this guide and tick the "Done" box for each step.
-
-We invite you to ask yourself questions, such as:
-
-- Does my vSphere managed on OVHcloud infrastructure use multi-vDC?
-    - Yes, I need to migrate all my data to one data centre in order to migrate to VCD.
-    - No, I'll move on to the next step.
-
-- Does my VMware on OVHcloud infrastructure use fault tolerance (FT)?
-    - Yes, so I have to disable it before migrating, otherwise I won't be able to migrate to VCD.
-    - No, I'll move on to the next step.
-
-- And so on...
-
-#### Is VCD compatible with PCI-DSS, SNC, HDS certifications?
-
-No, VCD on OVHcloud is not yet compatible with PCI-DSS, SNC, HDS security options.
-
-#### What type of migration will be used?
-
-This will be a hot cross vCenter migration.
+### Step 1 - Requirements and special cases (mandatory)
 
-#### Will the Dedicated Cloud be retained after the migration?
+What are the special use cases and solutions for your managed vSphere on OVHcloud to migrate to VCD? on OVHcloud ?
 
-No, all configurations and Dedicated Cloud data will be deleted after the migration.
+Once you have checked these requirements, your managed vSphere on OVHcloud VMs will be migrated (hot migration) fully and operated by the OVHcloud teams.
```

Comment:
"Once you have fulfilled these requirements, the OVHcloud teams will migrate your VMs from your vSphere on OVHcloud environment using a hot migration path."

Reply:
I have added : "Once you have fulfilled these requirements, the OVHcloud teams will migrate the VMs in the main vSphere datacentre using a hot migration path."
Let me know what you think

Reply:
This seems to be an old version, but your recommendations helped me to find a better phrasing